Library/Formula/gitman.rb

```diff
@@ -0,0 +1,14 @@
+class Gitman < Formula
+  homepage "https://github.com/jonstaff/gitman"
+  url "https://github.com/jonstaff/gitman/archive/v1.0.tar.gz"
+  version "1.0"
```

Comment:
Aye. The ` version ` line is safe to remove in this case - ` brew ` can automatically read a version from a relatively simple URL like this.